spec/models/manageiq/providers/redhat/infra_manager/refresh/refresher_target_vm_spec.rb

```diff
@@ -66,6 +66,32 @@
     assert_storage(storage, vm)
   end
 
+  context 'vm migration' do
+    before(:each) do
```

Comment:
> move this context into its own spec file

No, I meant wrapping the other tests in this file in their own context with their own `before` block and leave this test as is.
Now this `before` block gets executed and the one at the beginning of the file. And I think this is not what you want.